src/System.Text.Json/src/System/Text/Json/Serialization/JsonClassInfo.AddProperty.cs

```diff
@@ -174,7 +174,7 @@ internal JsonPropertyInfo CreateRootObject(JsonSerializerOptions options)
 
         internal JsonPropertyInfo CreatePolymorphicProperty(JsonPropertyInfo property, Type runtimePropertyType, JsonSerializerOptions options)
```

Comment:
I am also curios about why `JsonPropertyInfo` is being passed by `ref` when its a class (when looking at callers of `CreatePolymorphicProperty`). Isn't that redundant?

https://github.com/dotnet/corefx/blob/bfe528311e52fd41afca1c981dc28c2e62d96721/src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.Helpers.cs#L25-L37
cc @steveharter

Reply:
> Isn't that redundant?

It's assigning into it.  Presumably the caller of GetRuntimePropertyInfo needs the updated reference.